matching-service/const/constants.js

```diff
@@ -0,0 +1,12 @@
+exports.EVENTS = {
```

Comment:
Would singular "EVENT" be more appropriate?

Reply:
I agree, have changed it, thanks!